docs/best_practice/cluster_upgrade.txt

```diff
@@ -124,11 +124,9 @@ The ``crate`` process supports multiple signals (see :ref:`cli_signals`) that in
 
   sh$ kill -USR2 `cat /path/to/crate.pid`
 
-Depending on your installation you can make use of the init script.
-
-E.g. ::
+.. note::
 
-  sh$ service crate graceful-stop
+  If your system uses SysVinit (e.g. Debian Wheezy), you can also use ``sh$ service crate graceful-stop``. However, Upstart and systemd do not support this.
```

Comment:
Maybe give the user an other option what he could do when using Upstart and systemd. (if possible)